winlogbeat/checkpoint/checkpoint.go

```diff
@@ -45,6 +45,7 @@ type EventLogState struct {
 	Name         string    `yaml:"name"`
 	RecordNumber uint64    `yaml:"record_number"`
 	Timestamp    time.Time `yaml:"timestamp"`
+	Bookmark     string    `yaml:"bookmark"`
```

Comment:
Since this is optional for the eventlog API add `,omitempty` to the yaml tag.

```
update_time: 2018-01-24T16:05:31.2972263Z
event_logs:
- name: WinlogbeatTestPython
  record_number: 37134
  timestamp: 2018-01-24T16:05:30Z
  bookmark: ""
```

[source](https://beats-ci.elastic.co/job/elastic+beats+pull-request+multijob-windows/2929/beat=winlogbeat,label=windows/artifact/src/github.com/elastic/beats/winlogbeat/build/system-tests/run/test_eventlogging.Test.test_registry_data/data/.winlogbeat.yml/*view*/)

Reply:
done